Sources/StreamChat/StateLayer/Chat.swift

```diff
@@ -936,6 +942,51 @@ public final class Chat {
         try await channelUpdater.stopWatching(cid: cid)
     }
     
+    /// Subscribes to web-socket events of a single type in this channel.
+    ///
+    /// - Note: The handler is always called on the main thread.
+    /// - SeeAlso: ``ChatClient.subscribe(to:handler:)`` for subscribing to client events.
+    ///
+    /// - Parameters:
+    ///   - event: The event type to subscribe to (e.g. ``MessageNewEvent``).
+    ///   - handler: The handler closure which is called when the event happens.
+    ///
+    /// - Returns: A cancellable instance, which you use when you end the subscription. Deallocation of the result will tear down the subscription stream.
+    public func subscribe<T>(to event: T.Type, handler: @escaping (T) -> Void) -> AnyCancellable where T: Event {
+        eventNotificationCenter.subscribe(
+            to: event,
+            filter: { [cid] in
+                EventNotificationCenter.channelFilter(cid: cid, event: $0)
+            },
+            handler: handler
+        )
+    }
```

Comment:
Like I said above, I think we should actually remove this 👍 Unless I'm missing something? What the customer can do here, is the same of using directly the `ChatClient.subscribe(to: ChannelEvent.Type)` right?